app/core/core-ui/src/main/kotlin/com/hedvig/android/core/ui/SelectIndicationCircle.kt

```diff
@@ -12,21 +12,26 @@ import androidx.compose.material3.Surface
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.unit.dp
 import com.hedvig.android.core.designsystem.preview.HedvigPreview
 import com.hedvig.android.core.designsystem.theme.HedvigTheme
 import com.hedvig.android.core.ui.preview.BooleanCollectionPreviewParameterProvider
 
 @Composable
-fun SelectIndicationCircle(selected: Boolean, modifier: Modifier = Modifier) {
+fun SelectIndicationCircle(
+  selected: Boolean,
+  modifier: Modifier = Modifier,
+  selectedIndicationColor: Color = LocalContentColor.current,
+) {
   Spacer(
     modifier
       .size(24.dp)
       .clip(CircleShape)
       .then(
         if (selected) {
```

Comment:
Would be nice to animate this color :)

Reply:
added in the new PR:) not sure what our animation guidelines are, just used tween with 500 millis